instagram-app/src/components/CommentSection/CommentAdd.js

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+
+const CommentAdd = props =>{
+    return (
+        <div className="commentAdd">
+            <form onSubmit= {props.addComment}>
+                <input
+                    type="text"
+                    value={props.comment}
+                    placeholder="Add Comment..!"
+                    onChange={props.newComment}
+                />
```

Comment:
same here. I don't think when you're passing the props you are calling your onChange function the same thing. Double check the naming.